roles/lib_utils/src/test/unit/test_yedit.py

```diff
@@ -200,6 +200,7 @@ def test_append_twice_to_list(self):
         yed.append('x:y:z', [5, 6])
         yed.append('x:y:z', [5, 6])
         self.assertTrue(yed.get('x:y:z') == [1, 2, 3, [5, 6], [5, 6]])
+        # pylint: disable=maybe-no-member
```

Comment:
~~Currently the tests are not going through `pylint` (I'm changing that in #3511).
In our [Best Practices Guide](https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc#All-PyLint-rule-disables-MUST-be-documented-in-the-code) we say to always document pylint disables.~~

~~I volunteer to document this one if someone explain why it was added?~~

---

The line below, `self.assertTrue(2 == yed.get('x:y:z').count([5, 6]))` seems redundant after the line above `self.assertTrue(yed.get('x:y:z') == [1, 2, 3, [5, 6], [5, 6]])`.

If `x:y:z` is the given list, then we are just testing Python when we ask if the count of `[5, 6]` is two...

I'll go ahead and remove this redundant assert.